codice/ddf
# DDF-04401 Updated RTF transformer to remove `ext.` prefix

#### What does this PR do?
Updated RTF transformer to remove `ext.` prefix.  
#### Who is reviewing it? 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.
@Schachte @Bdthomson 

@jrnorth
@troymohl
#### How should this be tested?

Build / Install / Ingest something with extended attributes / Verify RTF export.
#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4401 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
